Fix core setup, Closes #224 (#225)
LiskHQ/lisk-docs: setup/docker.md

Clone the [Lisk Repository](https://github.com/LiskHQ/lisk-core). 

> ```bash

Reviewer: This will only work if you are `root`; we should (also?) give an example using `sudo` (`sudo -u lisk -i`) especially since we are using `sudo` above.

#### Option 1: Makefile

> We recommend to use the Makefile.

Reviewer: ```suggestion
We recommend using the Makefile.
```

Makefile provides a convenient way to [sync your node from snapshot](../administration/docker.md#sync-from-snapshot):

```bash

> make coldstart  # will download and sync from a recent blockchain snapshot for you

Reviewer: ```suggestion
make coldstart  # will download and restore from a recent blockchain snapshot for you
```

```

> **Note:** If you want to synchronize your node starting form the genesis block, it might take a significant amount of time until your local node will be fully syncronized with the blockchain network.
> We recommend to use "make coldstart" in case you want/need your node ready to use quickly.

```bash

> make  # will sync from genesis block on first start up

Reviewer: ```suggestion
make  # will sync from genesis block on first startup
```

docker-compose logs  # see logs
```

### Verify

As final step, verify your node is connected and in sync with the network, e.g. by asking about your nodes' status by using the API:

```bash

> docker exec <container> curl -X GET "http://localhost:7000/api/node/status" -H "accept: application/json"

Reviewer: ```suggestion
docker exec <container> curl http://localhost:7000/api/node/status --header "accept: application/json"
```

```

Where `<container>` is the docker container, Lisk Core is running in.

The result should look like this:

```json
{
  "meta": {},
  "data": {
    "broadhash": "ca930994bc1a6a92a47afb7310e3d9903f5e98ce56a6c5fdf444ba34f24c1543",
    "consensus": 94,
    "currentTime": 1558358294074,
    "secondsSinceEpoch": 94249094,
    "height": 8306047,
    "loaded": true,
    "networkHeight": 8306047,
    "syncing": false,
    "transactions": {
      "confirmed": 928836,
      "unconfirmed": 0,
      "unprocessed": 0,
      "unsigned": 0,
      "total": 928836
    }
  },
  "links": {}
}
```

> When your node is synced, the values of `networkHeight` and `height` should be (nearly) equal.

Reviewer: Maybe we can mention (testnet-)explorer.lisk.io as points of comparison?
